### ISSUE TITLE
remove Python 2.6 test in coverall.

### ISSUE THREAD

**Issue by shiodat** [jubatus/jubakit#99]
Coverall doesn't support Python 2.6.

```
$ pip install coverall
DEPRECATION: Python 2.6 is no longer supported by the Python core team, please upgrade your Python. A future version of pip will drop support for Python 2.6
Collecting coverall
/home/daats/.pyenv/versions/2.6.9/lib/python2.6/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
  SNIMissingWarning
/home/daats/.pyenv/versions/2.6.9/lib/python2.6/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Could not find a version that satisfies the requirement coverall (from versions: )
No matching distribution found for coverall
```

**Comment by hugovk** [comment]
Can this be closed now https://github.com/jubatus/jubakit/pull/100 has been merged?

**Comment by kmaehashi** [comment]
(I think package name is `coveralls`, not `coverall` ...) https://pypi.python.org/pypi/coveralls
Seems OK to close, anyway.

**Comment by rimms** [comment]
Fixed via #100 

Thanks you! @hugovk , @kmaehashi  .
